scrapers/nympho.yml

```diff
@@ -0,0 +1,20 @@
+# Be sure to set a User Agent in Settings! (ex:Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0)
```

Comment:
updated to include swallowed and trueanal
```yaml
# Be sure to set a User Agent in Settings! (ex:Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0)
name: nympho
sceneByURL:
  - action: scrapeXPath
    url:
      - nympho.com/view/
      - swallowed.com/view/
      - trueanal.com/view/
    scraper: sceneScraper
xPathScrapers:
  sceneScraper:
    scene:
      Title: //h2[@class="title"]/text()|//h1[@class="title"]/text()
      Details: //div[@class="content-page-info"]/div[@class="desc"]/p/text()
      Performers:
        Name: //div[@class="content-page-info"]//h4[@class="models"]/a
      Studio:
        Name:
          selector: //a[contains(@class,"navbar-brand")]/@href
          postProcess:
            - replace:
                - regex: .*tour\.(\w+).*
                  with: $1
            - map:
                nympho: Nympho
                trueanal: TrueAnal
                swallowed: Swallowed

      Image: //video[contains(@id,"ypp-player")]/@poster
# No Date, no Tags
# Last Updated September 20, 2020
```

Reply:
Added AllAnal as well 😃